frontend_app_file/src/container/File.jsx

```diff
@@ -89,6 +89,17 @@ class File extends React.Component {
     i18n.changeLanguage(this.state.loggedUser.lang)
 
     document.addEventListener(CUSTOM_EVENT.APP_CUSTOM_EVENT_LISTENER, this.customEventReducer)
+
+    const previouslyUnsavedComment = localStorage.getItem(
+      generateLocalStorageContentId(this.state.content.workspace_id, this.state.content.content_id, this.state.appName, 'comment')
+    )
+    if (previouslyUnsavedComment) this.setState({ newComment: previouslyUnsavedComment })
+
+    this.loadContent().then(() => {
+      this.loadTimeline()
+      this.buildBreadcrumbs()
+      if (this.state.config.workspace.downloadEnabled) this.loadShareLinkList()
+    })
```

Comment:
it is a bad practice to put async call to api on the constructor.
I cannot approve this without a good reason